qcodes/instrument_drivers/QDev/QDac.py

```diff
@@ -0,0 +1,473 @@
+# QCoDeS driver for QDac. Based on Alex Johnson's work.
+
+import time
+import visa
+import logging
+
+from datetime import datetime
+from functools import partial
+from time import sleep
+
+from qcodes.instrument.visa import VisaInstrument
+from qcodes.utils import validators as vals
+
+log = logging.getLogger(__name__)
+
+
+class QDac(VisaInstrument):
+    """
+    Driver for the QDev digital-analog converter QDac
+
+    Based on "DAC_commands_v_13.pdf"
+    Tested with Software Version: 0.160915
+
+    The driver assumes that the instrument is ALWAYS in verbose mode OFF
```

Comment:
So, I talked to Rikke today, and she already has a firmware upgrade, meaning that we can implement smooth ramping next week. Then the driver can be properly finished and give a huge WARNING when the firmware version is off.